lua/code_action_menu/lsp_objects/actions/code_action.lua

```diff
@@ -80,7 +80,24 @@ function CodeAction:execute()
   if self:is_workspace_edit() then
     vim.lsp.util.apply_workspace_edit(self.server_data.edit, 'utf-8')
   elseif self:is_command() then
-    vim.lsp.buf.execute_command(self.server_data.command)
+    local client = vim.lsp.get_client_by_id(self.client_id)
+    local fn = client.commands[self.server_data.command.command]
```

Comment:
In my tests, this is not correct and also not how the native implementation does it. According to the specification this is just `self.server_data.command`. Do you have a language server which formats that differently?

Reply:
Can you share your tests?

The navite implementation:

```lua
local fn = client.commands[command.command] or vim.lsp.commands[command.command]
```

https://github.com/neovim/neovim/blob/75ff156d9b11f23f6156cd6b90fb7bad0a83fd18/runtime/lua/vim/lsp/buf.lua#L512-L513